pgm-2.txt

```diff
@@ -0,0 +1,7 @@
+
+#!/bin/bash
+echo "hellooooooo"
+find *.log > z.txt
+find *.txt >> z.txt
+sudo mkdir /home/sudhir/zout
+tar -zczf matches.tar.gz /home/main
```

Comment:
This will create a tar file called matches.tar.gz, I want a tar named matches.tar and then the tar filed needs to be compressed to matches.tar.gz